service/create/validation.go

```diff
@@ -3,17 +3,31 @@ package create
 import (
 	"github.com/giantswarm/awstpr"
 	"github.com/giantswarm/awstpr/spec/aws"
+	"github.com/giantswarm/clustertpr/spec"
 	"github.com/giantswarm/microerror"
 )
 
-func validateWorkers(workers []aws.Node) error {
-	if len(workers) < 1 {
+func validateMasters(awsMasters []aws.Node, masters []spec.Node) error {
```

Comment:
Here the validation checks are extended. For masters currently we just check there is a single node.